otterdog/eclipse-set.jsonnet

```diff
@@ -44,6 +44,15 @@ orgs.newOrg('eclipse-set') {
       delete_branch_on_merge: true,
       has_wiki: false,
       web_commit_signoff_required: false,
+      branch_protection_rules: [
+        {
```

Comment:
you should use predefined functions to create resources as this would take default values into account. See an example here: https://otterdog.readthedocs.io/en/latest/reference/organization/repository/branch-protection-rule/

Reply:
Another real world example can be seen here: https://github.com/adoptium/.eclipsefdn/blob/main/otterdog/adoptium.jsonnet#L271

Reply:
ok, thank you for the tip